taskcluster/ci/build/kind.yml

```diff
@@ -5,7 +5,6 @@
 loader: taskgraph.loader.transform:loader
 
 transforms:
-    - mozillavpn_taskgraph.transforms.release_index:transforms
```

Comment:
This was accidentally leftover and is no longer being used as no `build` tasks are setting the `add-index-route` key.